addons/l10n_br/views/account_move_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_invoice_filter" model="ir.ui.view">
+        <field name="name">account.invoice.select.inherit</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_account_invoice_filter"/>
+        <field name="arch" type="xml">
+            <xpath expr="//search/group/filter[@name='status']" position="after">
+                <filter string="Adjustment Invoices" name="debit_notes_only" domain="[('debit_origin_id', '!=', False)]"/>
```

Comment:
Why?  This will appear for all debit notes and not only in the case of BR.  So, please remove it here and we can see to have it in generic separately.

Reply:
Removed. Rubi will launch a separate task.